common/lib/xmodule/xmodule/x_module.py

```diff
@@ -366,6 +366,14 @@ def display_name_with_default_escaped(self):
         return course_metadata_utils.display_name_with_default_escaped(self)
 
     @property
+    def tooltip_content(self):
+        """
+        Return a display name for use if this is the top-level block in a course unit.
+        Return the display name with default unless this block is not visible to students.
+        """
```

Comment:
Is this docstring accurate?  Did you intend to have both lines?  It seems to be describing the implementation rather than the semantics of the new method.